src/SolutionTemplate/UnoSolutionTemplate.net6/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -31,22 +31,47 @@
 		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
 	</ItemGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='net6.0-android'">
-		<PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
-		<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.35" />
-	</ItemGroup>
+	<Choose>
+		<When Condition="'$(TargetFramework)'=='net6.0-android'">
+			<ItemGroup>
+				<PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
+				<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.35" />
+			</ItemGroup>
+			<ItemGroup>
+				    <AndroidEnvironment Include="Android/environment.conf" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-ios'">
+		  	<PropertyGroup Condition="'$(TargetFramework)'=='net6.0-ios'">
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+			</PropertyGroup>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.3.0" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-maccatalyst'">
+			<PropertyGroup>
+				<!-- Configure the GC -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='net6.0-ios' or '$(TargetFramework)'=='net6.0-maccatalyst'">
-		<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.3.0" />
-	</ItemGroup>
+				<!-- Required for unknown crash as of .NET 6 Mobile Preview 13 -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --registrar:static</MtouchExtraArgs>
 
-	<PropertyGroup Condition="'$(TargetFramework)'=='net6.0-maccatalyst'">
-		<!-- Required for unknown crash as of .NET 6 Mobile Preview 13 -->
-		<MtouchExtraArgs>--registrar:static</MtouchExtraArgs>
+				<!-- Full globalization is required for Uno -->
+				<InvariantGlobalization>false</InvariantGlobalization>
+			</PropertyGroup>
 
-		<!-- Full globalization is required for Uno -->
-		<InvariantGlobalization>false</InvariantGlobalization>
-	</PropertyGroup>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.3.0" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-macos'">
+			<PropertyGroup>
+				<!-- Configure the GC -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
```

Comment:
That should not be needed (but likely won't hurt) as `net6.0-macos` uses the CoreCLR, not mono, JIT/GC.

Reply:
Thanks! TIL :)